src/test/unit/index.html

```diff
@@ -69,6 +69,7 @@
             "specific_feature_tests/class_toggling.js",
             "specific_feature_tests/links.js",
             "specific_feature_tests/dialogs.js",
+            "specific_feature_tests/native-edit-registration.js",
```

Comment:
All of the other test modules use underscores. Kind of a gross mix, I guess. Maybe something to fix later.